scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -148,6 +148,23 @@ def generate_run_numbers(run_number_string):
     return run_list
 
 
+def _generate_vanadium_name(vanadium_string, is_spline, *args):
```

Comment:
I don't know what is the documentation policy here, but most of the other functions in this file seem to have docstrings. Might be a good idea to have one here, as well.